configure.ac

```diff
@@ -22,6 +22,8 @@ AC_PROG_CC
 AC_PROG_INSTALL
 PKG_PROG_PKG_CONFIG
 
+CFLAGS="$CFLAGS -std=gnu99"
+
```

Comment:
quick google suggests we should be using `AC_PROG_CC_C99` here, right? or rather `AC_PROG_CC_STDC` though that's less specific.

Reply:
`AC_PROG_CC_STDC` doesn't pass `-std=gnu99` if compiler defaults to `-std=gnu11` which may lead to a regression due to discrepancy with Meson build.